templates/windows/HabService.dll.config.erb

```diff
@@ -13,5 +13,6 @@
     <add key="ENV_HAB_BLDR_URL" value="<%= @bldr_url %>" />
     <% end %>
     <add key="launcherArgs" value="--no-color <%= @exec_start_options %>" />
+    <add key="launcherPath" value="C:\Hab\pkgs\core\hab-launcher\13553\20200408005604\bin\hab-launch.exe"/>
```

Comment:
should this be hard coded?

Reply:
So, according to the habitat team, when launcher is started it will automatically overwrite this. I can add a test to prove that out if you like. I'd like to find a better way for sure.

Reply:
Will follow up since this pertains to your other question about this file